examples/apiproduct/dogs-cats/dogs.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dogs
+  labels:
+    svc: dogs
+spec:
+  selector:
+    matchLabels:
+      svc: dogs
+  template:
+    metadata:
+      labels:
+        svc: dogs
+    spec:
+      containers:
+        - name: dogs
+          image: quay.io/3scale/authorino:echo-api
```

I'd recommend instead using:

```suggestion
          image: quay.io/3scale/authorino-examples:talker-api
```